Makefile

```diff
@@ -17,58 +17,28 @@ endif
 UNAME_S := $(shell uname -s)
 UNAME_M := $(shell uname -m)
 
-ifeq ($(UNAME_S),Linux)
-LD           = lld -flavor link
-LIB          = llvm-lib
-AS           = clang
-CC           = clang
-CXX          = clang++
-ifneq ($(UNAME_M),x86_64)
-CGC          = $(NXDK_DIR)/tools/cg/linux/cgc.i386
-else
-CGC          = $(NXDK_DIR)/tools/cg/linux/cgc
-endif #UNAME_M != x86_64
-endif
-ifeq ($(UNAME_S),Darwin)
-LD           = /usr/local/opt/llvm/bin/lld -flavor link
-LIB          = /usr/local/opt/llvm/bin/llvm-lib
-AS           = /usr/local/opt/llvm/bin/clang
-CC           = /usr/local/opt/llvm/bin/clang
-CXX          = /usr/local/opt/llvm/bin/clang++
-CGC          = $(NXDK_DIR)/tools/cg/mac/cgc
-endif
 ifneq (,$(findstring MSYS_NT,$(UNAME_S)))
 $(error Please use a MinGW64 shell)
 endif
-ifneq (,$(findstring MINGW,$(UNAME_S)))
-LD           = lld-link
-LIB          = llvm-lib
-AS           = clang
-CC           = clang
-CXX          = clang++
-CGC          = $(NXDK_DIR)/tools/cg/win/cgc
-endif
+
+LD           = $(NXDK_DIR)/usr/bin/nxdk-link
+LIB          = $(NXDK_DIR)/usr/bin/nxdk-lib
+AS           = as
```

Comment:
This wasn't required yet, but should be fixed; this should also just be `clang` for now.